### PR TITLE
ao_pipewire: for_each_sink(): report errors

### PR DESCRIPTION
**Pull request by t-8ch**
Cc @philipl

### CODE REVIEW

File: audio/out/ao_pipewire.c [hunk 1]
```diff
@@ -185,7 +185,10 @@ static void on_param_changed(void *userdata, uint32_t id, const struct spa_pod *
         return;
     }
 
-    pw_stream_update_params(p->stream, params, 1);
+    if (pw_stream_update_params(p->stream, params, 1) < 0) {
+        MP_ERR(ao, "Could not update stream parameters\n");
+        return;
```
Comment (philipl) [inline audio/out/ao_pipewire.c:190]: No need `return`. It's already the end of the function.

**Review by philipl — COMMENTED** [review]
Just one nit.

File: audio/out/ao_pipewire.c (hunk 1 above)
Comment (t-8ch) [reply audio/out/ao_pipewire.c:190]: This was intentional to be consistent between the different error paths and make it explicit that we want to return even if we later extend this function.

If you prefer to remove it I can remove it though.

Comment (philipl) [reply audio/out/ao_pipewire.c:190]: Fair enough. let's leave as it then.

**Review by philipl — APPROVED** [review]

**Pull request merged**